app-arch/torcx/files/vendor.json

```diff
@@ -0,0 +1,11 @@
+{
+    "kind": "profile-manifest-v0",
+    "value": {
+        "images": [
+            {
+                "name": "docker",
+                "reference": "17.05.0"
```

Comment:
Okay, it makes sense to set the vendor reference to `com.coreos.cl` which would be a symlink to `17.05` for alpha.  Then when it rolls to beta, we can just point the symlink at `1.12` instead for Kubernetes support, for consistency with what we're doing now.